app/scripts/lib/uns-ipfs/setup.js

```diff
@@ -0,0 +1,49 @@
+import {
+    isValidUnstoppableDomainName,
+    getUdTlds,
+} from "../../../../ui/helpers/utils/util";
+import resolveUnsToIpfsContentId from "./resolver";
+import browser from 'webextension-polyfill';
+
+export default async function setupUnsIpfsResolver({
```

Comment:
not really, a lot of what we did was spin off the ENS implementation and they were passing in an object and we left one in there just in case. we can definitely remove it